docs/template_plugin/tests/functional/op_reference/logical.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "logical.hpp"
+
+namespace reference_tests {
+namespace LogicalOpsRefTestDefinitions {
+namespace {
+
+TEST_P(ReferenceLogicalLayerTest, LogicalWithHardcodedRefs) {
+    Exec();
+}
+
+}  // namespace
+}  // namespace LogicalOpsRefTestDefinitions
+}  // namespace reference_tests
```

Comment:
Please add missing new line at the end of files

Reply:
Done